src/nccl_ofi_rdma.c

```diff
@@ -1293,12 +1303,12 @@ static inline int handle_write_comp(struct fi_cq_tagged_entry *cq_entry,
 
 	ret = inc_recv_seg_completion(recv_segms_req, cq_entry->len, total_segms);
 	if (OFI_UNLIKELY(ret != 0)) {
-		return ret;
+
```

Comment:
This seems wrong.. it's an empty if.

Reply:
initial revision was a single commit, and in the process of splitting it into smaller commits I think I lost the transformation I was trying to make. Fixed in next commit.

Reply:
Please fix this in the same commit before this PR gets merged. We don't want to knowingly introduce bugs in one commit and fix them in another one.

Reply:
I meant fixed in next revision with a fixup, yes.